### PR TITLE
Feature/ndev 661

### PR DESCRIPTION
**Pull request by romanova-natasha**

### CODE REVIEW

File: integration/tests/economy/conftest.py [hunk 1]
```diff
@@ -19,3 +20,9 @@ async def get_price():
     price = loop.run_until_complete(get_price())
     with allure.step(f"SOL price {price}$"):
         return price
+
+
+@pytest.fixture(scope="function")
+def sol_client_tx_v2(pytestconfig: Config):
```
Comment (gigimon) [inline integration/tests/economy/conftest.py:26]: We have fixture with solana client https://github.com/neonlabsorg/neon-tests/blob/develop/integration/tests/conftest.py#L49
Why You need additional client from Pyth lib?

File: integration/tests/economy/test_economics.py [hunk 2]
```diff
@@ -853,3 +873,83 @@ def test_interact_with_contract_from_non_payed_user(self):
...
+
+        sol_balance_after = self.operator.get_solana_balance()
+        neon_balance_after = self.operator.get_neon_balance()
+
+        assert sol_balance_before > sol_balance_after_deploy > sol_balance_after
+        assert neon_balance_after > neon_balance_after_deploy > neon_balance_before
+        self.assert_profit(sol_balance_before - sol_balance_after, neon_balance_after - neon_balance_before)
+
+
+def wait_for_block(client, block, timeout=60):
```
Comment (gigimon) [inline integration/tests/economy/test_economics.py:942]: You can use simple synchronous solana client without asyncio to get block

**Review by gigimon — CHANGES_REQUESTED** [review]

File: integration/tests/economy/conftest.py (hunk 1 above)
Comment (romanova-natasha) [reply integration/tests/economy/conftest.py:26]: That was needed cause solana.rpc.api.Client does not have getBlock method with an opportunity to request transaction v2, it has a method 
```
def get_block(
        self,
        slot: int,
        encoding: str = "json",
    )
```
so I can request only v1 transaction.

Client method from pythclient.solana 
```
async def http_send(self, method: str, params: Optional[List[Any]] = None, *, return_error: bool = False)
```
allows me to make a request like:
```
{"jsonrpc": "2.0", "id": "3", "method": "getBlock", "params": [186454, {"encoding": "json", "transactionDetails": "full", "maxSupportedTransactionVersion": 2}]}
```
**maxSupportedTransactionVersion** is required for ALT

File: integration/tests/economy/test_economics.py (hunk 2 above)
Comment (romanova-natasha) [reply integration/tests/economy/test_economics.py:942]: see above

File: integration/tests/economy/conftest.py (hunk 1 above)
Comment (gigimon) [reply integration/tests/economy/conftest.py:26]: Please add comment for this and I will merge

Comment (romanova-natasha) [reply integration/tests/economy/conftest.py:26]: done

**Pull request merged**